Jucer2Reprojucer/main.cpp

```diff
@@ -1619,6 +1622,8 @@ int main(int argc, char* argv[])
                                       return "(default)";
                                     if (value == "stdcpp14")
                                       return "C++14";
+                                    if (value == "stdcpp17")
+                                      return "C++17";
```

Comment:
The Visual Studio exporter setting "C++ standard to use" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-2cd603f4b77a06ba4f2d589cda677923L1863-L1865. At that time, Projucer didn't know about C++17 at all, so these 2 lines are actually dealing with unexpected input. Please remove them.